app/controllers/tile_metadata_controller.rb

```diff
@@ -19,6 +19,16 @@ def metadata
 
   private
 
+    def generate_mosaic?
+      if resource.is_a?(RasterResource) && resource.decorate.raster_set?
+        true
+      elsif resource.is_a?(ScannedMap) && resource.decorate.map_set?
```

Comment:
Won't this give a false positive for map sets that don't have any raster children?

Reply:
Also can we just generate this for all scanned maps/rasters? If it's a scanned map with no raster children will something blow up?

Reply:
It won't blow up, just returns an error. To me, seems better than running the mosaic file_set query multiple times per request. We could rescue that error and return a 404.

Reply:
Fixed